src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -1,7 +1,9 @@
 const theme = {
-  white: '#FFFFFF',
+  breakpoints: [768, 992, 1200],
```

Comment:
This is gonna break grid-emotion

Reply:
Hmm break it how? Don't we want these as our breakpoints?

Reply:
I think it needs to be specified in a different format.

Things like this won't work 
anymore - https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/organizationDashboard/index.jsx#L50

Reply:
Ah it needs units, fixed.